### PR TITLE
refactor(home): modifying images for the home page

### PR DESCRIPTION
**Pull request by mfrachet**
https://github.com/Zenika/zenika.ca/issues/277

### CODE REVIEW

File: components/modules/Contact.vue [hunk 1]
```diff
@@ -95,6 +95,7 @@ export default {
 #contact-info {
   color: $text;
   display: flex;
+  font-size: 1.5em;
```
Comment (lbonanni) [inline components/modules/Contact.vue]: Why did you change the font size ? it looks very big this way 😕

Comment (mfrachet) [reply components/modules/Contact.vue]: Because white on white is not visible
